scripts/pre_commit_linter.py

```diff
@@ -1049,6 +1049,53 @@ def _check_docstrings(all_files):
     return summary_messages
 
 
+def _check_controllers(all_files):
```

Comment:
Please change this to `_check_controller_class_names` so that it's a bit more specific.